src/utilities/common.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Common shell function for the IMI
+# Functions available in this file include:
+#   - imi_failed 
+#   - run_inversion 
+
+
+# Description: Print error message for if the IMI fails
+#   Copy output file to output directory if it exists
+# Usage:
+#   imi_failed
+imi_failed() {
+    file=`basename "$0"`
+    printf "\nFATAL ERROR on line number ${1} of ${file}: IMI exiting."
+    if [ -d "${OutputPath}/${RunName}" ]; then
+        cp "${InversionPath}/imi_output.log" "${OutputPath}/${RunName}/imi_output.log"
+    fi
+    exit 1
+}
+
+# Description: remove sbatch memory and cpu resources
+# remove time too on AWS
+# Usage:
+#   remove_sbatch_headers runscript.run
+remove_sbatch_headers() {
```

Comment:
Is there a reason we need this? Can we simply remove those lines from the relevant run scripts (e.g. [ch4_run.template](https://github.com/geoschem/integrated_methane_inversion/blob/main/src/geoschem_run_scripts/ch4_run.template))?